Tensile/Parallel.py

```diff
@@ -29,15 +29,15 @@
 
 def CPUThreadCount(enable=True):
   from .Common import globalParameters
-  if not enable or globalParameters["CpuThreads"] == 0:
-    return 0
+  if not enable:
+    return 1
   else:
     if os.name == "nt":
       cpu_count = os.cpu_count()
     else:
       cpu_count = len(os.sched_getaffinity(0))
     cpuThreads = globalParameters["CpuThreads"]
-    if cpuThreads < 0:
+    if cpuThreads < 1:
         return cpu_count*abs(cpuThreads)
```

Comment:
I think this returns 0 if cpuThreads=0.
If we would like to make 0 behave like -1, it is better to change to "return cpu_count".

Reply:
Good catch, just posted a fix. Thanks!